recipes/oatpp-libressl/all/test_package/conanfile.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class OatppLibresslTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def build(self):
```

Comment:
The option `fPIC` does not do anything to the test_package as it builds only an executable,
and I don't think conan knows about `-fPIE` (yet).

```suggestion
    generators = "cmake", "cmake_find_package"

    def build(self):
```